osu.Game/OsuGame.cs

```diff
@@ -803,6 +805,9 @@ protected override void LoadComplete()
             loadComponentSingleFile(wikiOverlay = new WikiOverlay(), overlayContent.Add, true);
             loadComponentSingleFile(skinEditor = new SkinEditorOverlay(ScreenContainer), overlayContent.Add, true);
 
+            if (!args?.Any(a => a == @"--no-version-overlay") ?? true)
+                loadComponentSingleFile(versionManager = new VersionManager { Depth = int.MinValue }, ScreenContainer.Add);
```

Comment:
Loading the version manager single-file this late makes it so that all preceding single-file components block the version manager from showing and therefore it fades in later than it does on `master`, kind of ruining the intro.

See following video:

https://user-images.githubusercontent.com/20418176/149663431-d8c49f8a-17a9-4a16-a919-072e3543236a.mp4

The version overlay fades in mid-intro, while on `master` it is fully faded in before the intro even starts playing.

Reply:
Decided to load it first, before the logo, since it'll get show once the Loader from `CreateLoader()` loads the intro sequence.